rdbms-image/devicehive-backend/Dockerfile

```diff
@@ -1,12 +1,17 @@
-FROM frolvlad/alpine-oraclejdk8
+FROM picoded/ubuntu-openjdk-8-jdk
```

Comment:
Motivation was to use netcat ubuntu options for startup order. And OpenJDK Ubuntu was the most lightweight image I found. But in the latest commit I achieved the same behavior with netcat-openbsd for linux, so for saving disk space I returned to Alpine.